docs/serverless/02-01-serverless.md

```diff
@@ -12,19 +12,21 @@ Serverless relies on [Knative Serving](https://knative.dev/docs/serving/) for de
 
     >**NOTE:** Function Controller sets the Node.js 12 runtime by default.
 
-2. Function Controller (FC) detects a new Function CR.
+2. Before the Function can be saved or updated, it is verified sequentially by the [defaulting and validation webhooks](#supported-webhooks).
```

Comment:
Function is verified only by validation webhook, not by defaulting. Defaulting adjusts/updates/mutates missing/optional  fields.